boards/arm/beagle_bcf/beagleconnect_freedom.dts

```diff
@@ -127,16 +118,25 @@
 		reg = <0x4>;
 	};
 
-	light: opt3001-light@44 {
```

Comment:
It looks like this was in the wrong place. It should go under the sens_i2c or i2c0s I think it was called on other versions.